### PR TITLE
Feature/bri va

### PR DESCRIPTION
**Pull request by askur87**
add feature Bank Transfer BRI VA

### CODE REVIEW

File: demo/src/main/java/com/midtrans/demo/AppUtils.java [hunk 1]
```diff
@@ -102,6 +104,10 @@ public static PaymentMethodsModel getVaPaymentMethods(Context context, String ty
                     model = new PaymentMethodsModel(PAYMENT_NAME_BNI_VA, null, 0, "", 5, status);
                     break;
 
+                case PaymentType.BRI_VA:
+                    model = new PaymentMethodsModel(PAYMENT_NAME_BRI_VA, null, 0, "", 5, status);
```
Comment (farhanpahlevi) [inline demo/src/main/java/com/midtrans/demo/AppUtils.java:108]: Is the priority same with BNI?

File: uikit/src/main/java/com/midtrans/sdk/uikit/PaymentMethods.java [hunk 2]
```diff
@@ -155,6 +155,10 @@ public static BankTransfer createBankTransferModel(Context context, String type,
                     bankTransfer = new BankTransfer(type, context.getString(R.string.bni_bank_transfer), R.drawable.ic_bni, 4, context.getString(R.string.payment_bank_description_bni), status);
                     break;
 
+                case PaymentType.BRI_VA:
+                    bankTransfer = new BankTransfer(type, context.getString(R.string.bri_bank_transfer), R.drawable.ic_bri, 4, context.getString(R.string.payment_bank_description_bri), status);
```
Comment (farhanpahlevi) [inline uikit/src/main/java/com/midtrans/sdk/uikit/PaymentMethods.java:159]: Is this have same priority with BNI? (4)

**Review by farhanpahlevi — CHANGES_REQUESTED** [review]

File: demo/src/main/java/com/midtrans/demo/AppUtils.java (hunk 1 above)
Comment (askur87) [reply demo/src/main/java/com/midtrans/demo/AppUtils.java]: updated the priority please re-check @farhanpahlevi

File: uikit/src/main/java/com/midtrans/sdk/uikit/PaymentMethods.java (hunk 2 above)
Comment (askur87) [reply uikit/src/main/java/com/midtrans/sdk/uikit/PaymentMethods.java]: updated the priority please re-check @farhanpahlevi

**Comment by farhanpahlevi** [comment]
LGTM!

**Review by farhanpahlevi — APPROVED** [review]

**Comment by femmerling** [comment]
LGTM
LGTM

**Review by femmerling — APPROVED** [review]

**Pull request merged**